docs/introduction/entities.md

```diff
@@ -249,6 +356,16 @@ class BrandData
 }
 ```
 
+### Data for multidomain or multilanguage field
+
+As you can see in example above, multidomain or multilanguage field is an array.
+
+Multidomain field like `seoH1s` have to be indexed by `domainId` - integer ID of the given domain.
+Data factory have to prepare this array to contain keys for all domain IDs, because domain entities are created by these array items (even if their value is null).
```

Comment:
```suggestion
Data factory has to prepare this array to contain keys for all domain IDs, because domain entities are created by these array items (even if their value is null).
```